### PR TITLE
[Next.js] Add environment variable to allow disable of sitemap fetch in getStaticPaths

### PR DESCRIPTION
**Pull request by illiakovalenko**
## Description / Motivation
* Disable pages pre-render based on environment variable `DISABLE_SSG_FETCH` value

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Review by ambrauer — CHANGES_REQUESTED** [review]
Let's stick with `DISABLE_SSG_FETCH`. Related work has already started using this name and these need to be in sync.

File: packages/create-sitecore-jss/src/templates/nextjs/.env [hunk 1]
```diff
@@ -42,6 +42,11 @@ DEFAULT_LANGUAGE=
 # The way in which layout and dictionary data is fetched from Sitecore
 FETCH_WITH=<%- fetchWith %>
 
+# Indicates whether SSG `getStaticPaths` pre-render any pages
+# Set the environment variable DISABLE_SSG_STATIC_PATH_FETCH=true
+# to enable full ISR (Incremental Static Regeneration) flow, pre-fetch static paths
```
Comment (ambrauer) [inline packages/create-sitecore-jss/src/templates/nextjs/.env:47]: Not sure what ", pre-fetch static paths" at the end means. Remove it?
```suggestion
# to enable full ISR (Incremental Static Regeneration) flow
```

Comment (illiakovalenko) [reply packages/create-sitecore-jss/src/templates/nextjs/.env:47]: Wanted to describe a bit which functionality will be disabled, but it's fine to remove it

**Review by ambrauer — APPROVED** [review]
Looks good!

**Pull request merged**